packages/core/src/Utils/Datadog/index.ts

```diff
@@ -25,21 +25,30 @@ if (isProduction) {
     dataDogEnv = 'staging';
 }
 
-datadogRum.init({
-    applicationId: isStaging || isProduction ? DATADOG_APP_ID : '',
-    clientToken: isStaging || isProduction ? DATADOG_CLIENT_TOKEN : '',
-    site: 'datadoghq.com',
-    service: serviceName,
-    env: dataDogEnv,
-    sessionSampleRate: dataDogSessionSampleRate,
-    sessionReplaySampleRate: dataDogSessionReplaySampleRate,
-    trackUserInteractions: true,
-    trackResources: true,
-    trackLongTasks: true,
-    defaultPrivacyLevel: 'mask-user-input',
-    version: dataDogVersion,
-    trackFrustrations: true,
-    enableExperimentalFeatures: ['clickmap'],
-    excludedActivityUrls: [/^https:\/\/api.telegram.org.*$/],
-});
-datadogRum.startSessionReplayRecording();
+try {
+    datadogRum.init({
```

Comment:
```suggestion
    if(isProduction || isStaging)
      datadogRum.init({
```
what if we just add an if condition on top of it?